docs/gathered-data.md

```diff
@@ -436,6 +421,18 @@ Relevant Kubernetes API docs:
 Location in archive: config/pod/{namespace}/{pod-name}.json
 
 
+## SAPVsystemIptablesLogs
```

Comment:
BTW, I notice that this documentation was missing, so... I generate it again, and... here we have it.